digid_eherkenning/views/eherkenning.py

```diff
@@ -7,19 +7,30 @@
 from django.utils.translation import gettext as _
 from django.views.generic.base import TemplateView, View
 
+from ..choices import AssuranceLevels
 from ..exceptions import SAML2Error, eHerkenningNoRSINError
 from ..forms import SAML2Form
-from ..saml2.eherkenning import (
-    eHerkenningClient,
-    generate_dienst_catalogus_metadata,
-    generate_eherkenning_metadata,
-)
+
+# These two unused imports may be imported from here by 3rd party client code
+# ignoring F401 for now
```

Comment:
do we actually use that now? Can you maybe add a TODO so that we can get rid of them in the next (major) release?

Reply:
Open-formulieren doesn't and I doubt anyone does. The most comfortable way is to use the management commands or the admin views. It looks plausible these imports were only ever used in the previous views that were refactored to CBV.

Just noticed this is 0.7.0 so we *could* just remove them...

Reply:
You have my blessing - it's probably a left-over from the process to expose metadata via views, I had to do *a lot* of refactoring and probably just missed these.

How badly does flake complain? Can we add it to the CI checks yet in https://github.com/maykinmedia/django-digid-eherkenning/blob/master/.github/workflows/code_quality.yml#L23?